module3/lessons/Week4/LinkedIn.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: page
+title: Building an Effective LinkedIn Profile
+---
+
+### Learning Goals
+By the end of this lesson, you should have accomplished these learning goals:
+* Understand how to tell your story through LinkedIn
+* Know how to build a working draft of your LinkedIn profile with these components:
+  * Headshot / Cover Photo
+  * Headline updated to include your software identity
+  * Summary section to tell your professional story
+  * Experience
+
+### Section 1: Introduction
+
+❓What do you already know about LinkedinIn? Have you used it before?
+
+LinkedIn is a social media tool you can use to:
+* Tell your career story on digital platforms
+* Communicate with recruiters, peers, and thought-leaders in your industry
+* Connect with people to build your network
+* Search for and apply to jobs online
+* Research companies, employees, and available opportunities
+
+<section class="call-to-action" markdown="1">
+Create a [Linkedin](https://www.linkedin.com/feed/) Profile if you don't yet have one. Then find a LinkedIn profile that you like. What's one thing you like about it? 

```

Comment:
For some reason, this link isn't rendering correctly. Also, remove `/feed/` from the link